aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -1592,6 +1680,100 @@ resource "aws_elasticsearch_domain" "test" {
 `, randInt)
 }
 
+func testAccESDomainConfig_AdvancedSecurityOptionsUserDb(randInt int) string {
+	var user = ""
+	var options = `
+advanced_security_options {
+	enabled = true
+	internal_user_database_enabled = true
+	master_user_options {
+		master_user_name = "testmasteruser"
+		master_user_password = "Barbarbarbar1!"
+	}
+}`
+
+	return fmt.Sprintf(`
+%s
+
+resource "aws_elasticsearch_domain" "test" {
```

Comment:
lets flatten this to a single config and not pass the options (and redundant empty `var user = ""` ), it will reading the config easeir